webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -0,0 +1,93 @@
+.ldap-wizard-wrapper {
+    background-color: rgba(var(--center-channel-color-rgb), 0.04);
+
+    .ldap-wizard-content-wrapper {
+        display: flex;
+        overflow: hidden; // Prevent double scrollbars on the body if content overflows
```

Comment:
This makes me think we have something wrong with how we're nesting divs in the system console. I tried removing it, but the way that the whole system console page scrolls including the header and save bar when it's removed leads me to think our flexboxes are nested wrong.

I don't have any better suggestions for how to do it though, so nothing to change here now unless you want to try to unravel all that 😅

Reply:
Hah, I'd love to, but I battled with this for a bit to make it look good, I'm kind of leaving it as is for fear of making it (or the other pages) worse :)

Reply:
That's totally fine. Something definitely seems messed up with this nesting and scrolling, but it's not worth looking into right now